gobblin-docker/gobblin-wikipedia/ubuntu-gobblin-0.7.0/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gobblin/gobblin-distributions:ubuntu-gobblin-0.7.0
+
+ENV GOBBLIN_WORK_DIR /home/gobblin/work-dir
+
+RUN echo "Downloading Gobblin Wikipedia Configuration Files"
+RUN mkdir -p /etc/opt/gobblin/job-conf
+
+# Downloading wikipedia.pull
+RUN curl -L --progress-bar https://raw.githubusercontent.com/linkedin/gobblin/master/gobblin-example/src/main/resources/wikipedia.pull -o /etc/opt/gobblin/job-conf/wikipedia.pull
```

Comment:
Thanks for the changes Sahil. Can you also please create a placeholder github issue for adding release tag to gobblin-cli.properties download, it might be a while before we make a 0.9.0 release and we will forget to update it by then